doc/example/example_ordinal.ipynb

```diff
@@ -0,0 +1,1047 @@
+     "text": [
+      " 50%|█████     | 5/10 [00:07<00:06,  1.22s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 45\n",
+      "         Function evaluations: 91\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 60%|██████    | 6/10 [00:08<00:05,  1.25s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 58\n",
+      "         Function evaluations: 109\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 70%|███████   | 7/10 [00:10<00:03,  1.30s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 68\n",
+      "         Function evaluations: 132\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 80%|████████  | 8/10 [00:11<00:02,  1.26s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 54\n",
+      "         Function evaluations: 102\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 90%|█████████ | 9/10 [00:12<00:01,  1.35s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 81\n",
+      "         Function evaluations: 147\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 10/10 [00:14<00:00,  1.40s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 60\n",
+      "         Function evaluations: 110\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "np.random.seed(n_starts)\n",
+    "problem.objective.calculator.inner_solver = OptimalScalingInnerSolver(\n",
+    "    options={\n",
+    "        'method': 'reduced',\n",
+    "        'reparameterized': True,\n",
+    "        'intervalConstraints': 'max',\n",
+    "        'minGap': 1e-10,\n",
+    "    }\n",
+    ")\n",
```

Comment:
Maybe better to provide these `options` to `PetabImporter(..., ordinal=True, inner_solver_options=...)` so that the user doesn't need to import `OptimalScalingInnerSolver` and do these additional steps

Reply:
Ah, I see `PetabImporter.create_objective` takes this as input, I guess it would be good to allow users to also specify in `__init__` as above, for simpler code.

Reply:
Also, for the notebook, maybe better to run with default options in the first case, then import `OptimalScalingInnerSolver` later to show how it can be customized in the next cases

Reply:
Fine as is, if you don't want to showcase default behavior

Reply:
Yes, I was thinking for some time about where it makes the most sense to give the options, so I decided at objective, as it's used there directly. But it can be still changed post-objective creation, as shown in the notebook, by creating a new OptimalScalingInnerSolver basically.

In which __init__ would it be nice to be able to give them as well? 

I agree on the notebook, I'll include the different ways of construction.

Reply:
I meant in `PetabImporter.__init__`, since that's where users can also specify e.g. `hierarchical=True, ordinal=True` etc.

It can make sense to also accept this at other points, e.g. `PetabImporter.create_objective` as you have done, so that users can override/fine-tune/perform complicated experiments easier. But otherwise default to a `self.inner_solver_options` that is created in `PetabImporter.__init__`.

Anyway, up to you, this is a minor point, you've already designed it so users can specify things easily, nice job :+1:

Reply:
Ohh that's what you mean. Yes, good idea. Will add.

I added it now as `self._inner_solver_options` to the importer init. 
I also removed inner_problem_method, because it was given in the solver options, so now, only solver options need to be passed.